slidez-fe-core/src/pages/dashboard/UserProfile.tsx

```diff
@@ -81,15 +167,15 @@ const UserProfile = () => {
     }
 
     const handleLogo = () => {
-        if (userFirstName && userLastName) {
+        if (userData.firstName && userData.lastName) {
             setLogo(
-                userFirstName.charAt(0).toUpperCase() +
-                    userLastName.charAt(0).toUpperCase()
+                userData.firstName.charAt(0).toUpperCase() +
+                    userData.lastName.charAt(0).toUpperCase()
             )
-        } else if (userEmail) {
+        } else if (userData.email) {
             setLogo(
-                userEmail.charAt(0).toUpperCase() +
-                    userEmail.charAt(1).toUpperCase()
+                userData.email.charAt(0).toUpperCase() +
+                    userData.email.charAt(1).toUpperCase()
             )
         }
     }
```

Comment:
Create logo component somewhere, which will get firstName, lastName, email and return this circle based on that data

Reply:
Again

Reply:
I moved this code to separate file and import it when used. I thought that I will not create component logo, because otherwise I will have to  override most stile classes, it is wrong?